src/database/queries/delete_events_querie.js

```diff
@@ -0,0 +1,14 @@
+const Events = require('../event_schema');
+
+const deleteQuery = (req, callback) => {
+
+  Events.findByIdAndRemove(req, (err, response) => {
+    if (err) return res.status(500).send(err);
+    callback();
```

Comment:
I think here we need to give the callback ```(null, response)``` as parameters

Reply:
@NoureldeanSaed @FarahZaqout done :+1: